transport-native-epoll/src/main/java/io/netty/channel/epoll/Native.java

```diff
@@ -39,7 +39,7 @@
  *
  * <strong>Internal usage only!</strong>
  */
-final class Native {
+public final class Native {
```

Comment:
I guess we can do this once we start to add a kqueue transport. I would just keep it for now.